cdc/owner/schema.go

```diff
@@ -220,7 +223,14 @@ func (s *schemaWrap4Owner) BuildDDLEvents(
 		if err != nil {
 			return nil, errors.Trace(err)
 		}
-		event.FromJob(job, preTableInfo)
+		if job.BinlogInfo != nil && job.BinlogInfo.TableInfo != nil {
```

Comment:
can you add a comment about what situation will the job.BinlogInfo is nil?

Reply:
Done.